src/Bridges/indicatorbridge.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    IndicatorActiveOnFalseBridge{T}
+
+The `IndicatorActiveOnFalseBridge` replaces an indicator constraint activated on 0
+with the constraint activated on 1, creating an additional variable.
+"""
+struct IndicatorActiveOnFalseBridge{F <: MOI.AbstractScalarFunction, S <: MOI.AbstractScalarSet, T <: Real} <: AbstractBridge
```

Comment:
The type `T` should be first. Why should it be `Real` ? You can leave it without any supertype as it is done for other bridges

Reply:
I thought the `T` from `MOI.EqualTo` was constrained to `Real`, but it is on `Number` only

Reply:
No, for `MOI.EqualTo`, it can be on complex numbers too.